test/helpers.bash

```diff
@@ -198,6 +198,11 @@ function crictl() {
     "$CRICTL_BINARY" -r "unix://$CRIO_SOCKET" -i "unix://$CRIO_SOCKET" "$@"
 }
 
+# Run the runtime binary with the specified RUNTIME_ROOT
+function runtime() {
+    "$RUNTIME_BINARY_PATH" --root "$RUNTIME_ROOT" "$@"
```

Comment:
Hmm, did you mean `"$CONTAINER_RUNTIME"` here? There's no `"$RUNTIME_BINARY_PATH"` (yet?)

Reply:
Ah, I see, the order of commits seems to be wrong (or github is showing them in the wrong order), as this ^^^ belongs to the first commit, and `$RUNTIME_BINARY_PATH` is introduced by the third commit.

Reply:
oopsies, updated!